server/models/forecast.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import Column, DateTime, Integer, String
```

Comment:
Add `# type: ignore` here and anywhere else we import `sqlalchemy`. ([ref](https://github.com/clairBuoyant/server/pull/44/files#r950733473))

Run `poetry run check` to make sure everything is ok.

Reply:
Sounds good, it's not in this branch any longer though. I moved it to it's own respective branch for issue 34.